index.js

```diff
@@ -68,7 +68,46 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+
+header.addEventListener('click', () => {
+  toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
+
+likeBtn.addEventListener('click', () => {
+  const likes = parseInt(pLike.textContent)
+  pLike.textContent = likes + 1 + " Likes"
+})
```

Comment:
Looks good! The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. Nice work!